packages/mesh-browser-lite/src/types.ts

```diff
@@ -186,6 +186,9 @@ export interface Config {
     // Offers the ability to use your own web3 provider for all Ethereum RPC
     // requests instead of the default.
     web3Provider?: SupportedProvider;
+    // The maximum number of bytes per second that a peer is allowed to send before
+    // failing the bandwidth check. Defaults to 1 MiB.
```

Comment:
Can you double check to make sure there aren't any other outdated comments?

```suggestion
    // failing the bandwidth check. Defaults to 5 MiB.
```